Index: debian/changelog
===================================================================
--- debian/changelog	(revision 1044)
+++ debian/changelog	(working copy)
@@ -1,4 +1,4 @@
-mapcatcher (0.7.5.0-1) unstable; urgency=low
+mapcatcher (0.7.5.0-1ubuntu1) maverick; urgency=low
 
   * Initial release (closes: #589254)
 
Index: installer/build_gz.sh
===================================================================
--- installer/build_gz.sh	(revision 1045)
+++ installer/build_gz.sh	(working copy)
@@ -188,4 +188,4 @@
 rm -rf ../temp/
 
 # cd $MAINDIR/debian_support
-# dput ppa:ubunt-u-markbenjamin/mapcatcher $pkgname-1_source.changes
+# dput ppa:ubunt-u-markbenjamin/mapcatcher $pkgname-1ubuntu1_source.changes
